package.json

```diff
@@ -3,7 +3,29 @@
   "title": "jQuery",
   "description": "JavaScript library for DOM operations",
   "version": "4.0.0-pre",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "node": {
+        "module": "./dist/jquery.mjs",
+        "import": "./dist/jquery-node-wrapper.mjs",
+        "default": "./dist/jquery.js"
+      },
+      "browser": "./dist/jquery.mjs"
+    },
+    "./slim": {
+      "node": {
+        "module": "./dist/jquery.slim.mjs",
+        "import": "./dist/jquery-node-wrapper.slim.mjs",
+        "default": "./dist/jquery.slim.js"
+      },
+      "browser": "./dist/jquery.slim.mjs"
+    },
+    "./src/*.js": "./src/*.js",
+    "./amd/*.js": "./amd/*.js"
+  },
   "main": "dist/jquery.js",
+  "module": "dist/jquery.mjs",
```

Comment:
Parcel doesn't support `package.json` `exports` so this field is still needed:
https://github.com/parcel-bundler/parcel/issues/4155